app/views/admin/cms/collections/_form.html.erb

```diff
@@ -1,5 +1,7 @@
-<%= form_for [:admin, :cms, @resource] do |f| %>
-  <%= render "admin/cms/shared/errors" %>
+<%= form_for [:admin, :cms, @collection] do |f| %>
```

Comment:
i thought you had to set the `multipart` option to `true` for file uploads from forms to work?